docs/api.md

```diff
@@ -3,4 +3,4 @@
 The most important parameter for all endpoints except the `Home#index` is the
```

Comment:
absolutely, but current implementation doesn't support that, though as it can only be via the header. However, I think it may come down to the `FE` guys, i.e. what options does `react` provide?